!: edit page (#10663)
tldr-pages/tldr: pages/linux/!.md

`!{{number}}`

> - Substitute with a command that was used specified number of lines back:

Reviewer: Thanks for your contribution.

How about this?

```suggestion
- Substitute with a command that uses a specified number of lines back:
```

Author: I feel like that's not grammatically correct.

Reviewer 2: I'm pretty sure it's grammatically correct, but I'd interpret it to mean something different.

My suggestions would be:

```suggestion
- Substitute with a command that was used a specified number of lines back:
```

```suggestion
- Substitute with a command from a specified number of lines back:
```

> - Substitute with a command that was used specified number of lines back:

Reviewer 3: Well maybe this:
```suggestion
- Substitute with a command that is "number" in history:
```
